ci/pipelines/meta/deploy_meta.sh

```diff
@@ -130,7 +130,6 @@ YML
     echo "Target ${FLY_TARGET} already exists."
   fi
 
-  FLY=${FLY:-$(which fly)}
```

Comment:
since deploy_meta.sh is sometimes run by humans, why remove this?

Reply:
Because lines 119-120 make it never take effect, and rectifying that logic is beyond the scope of my change here.